lib/ymlr_test.exs

```diff
@@ -31,7 +29,38 @@ defmodule YmlrTest do
           name: testing-cc
       """
 
-      assert MUT.document!(input) == output
+      input = YamlElixir.read_from_file!("test_support/fixtures/iam_policy.yaml")
+      output = MUT.document!(input)
+
+      assert output == expected_output
+      assert YamlElixir.read_from_string!(output) |> MUT.document!() == output
+    end
+
+    test "k8s resource with atoms options" do
+      expected_output = """
+      ---
+      :apiVersion: iam.cnrm.cloud.google.com/v1beta1
+      :kind: :IAMPolicy
+      :metadata:
+        :annotations:
+          :cnrm.cloud.google.com/deletion-policy: abandon
+        :name: sa-testing-cc-iampolicy
+      :spec:
+        :bindings:
+          - :members:
+              - user:test@example.com
+            :role: roles/iam.serviceAccountUser
+        :resourceRef:
+          :apiVersion: iam.cnrm.cloud.google.com/v1beta1
+          :kind: :IAMServiceAccount
+          :name: testing-cc
+      """
```

Comment:
@ryanwinchester do you have a better example of a yaml doc with atom keys? because I'm pretty sure this is not a valid kubernetes resource / manifest, i.e. the cluster would probably complain if we tried to `kubectl apply` it (I did not try this)

Reply:
actually, this would really help me understand what the use case is

Reply:
The kubernetes file with atoms in the test is just a copy of your other test, but with atoms added in some places, in order to verify the atoms option works as intended. It is  not meant to be used with kubernetes. 

For use-cases, we're using it to write files that `YamlElixir` will later read. However, we require some keys and values to be atoms. `YamlElixir` has an `atoms: true` option that will parse values that should be atoms, as atoms. So, I just added the same option to `Ymlr` so that they can work together.

Right now I'm using the `Yamlr` fork with this feature.